src/main/webapp/WEB-INF/tags/instructor/results/copyPasteModal.tag

```diff
@@ -17,11 +17,10 @@
   <div class="modal-dialog modal-lg">
     <div class="modal-content">
       <div class="modal-header">
-        <span class="help-block" style="display:inline;">
-          Tips: After selecting the table, <kbd>Ctrl + C</kbd> to COPY and
-          <kbd>Ctrl + V</kbd> to PASTE to your Excel Workbook.
+        <span class="help-block">
```

Comment:
What are the effects of removing `display:inline`?

Reply:
display:inline was being used to override the display:block that this element gets from the help-block CSS class. This was to make sure that the tip text and buttons appeared on the same row.

Since I was moving the buttons below the now longer tip text, this inline styling could be removed. Also looking at your linked HTML style guide, inline styling should be avoided when possible and with my changes inline styling is no longer necessary. 

Also, I understand that there might be some reason why copyPasteModal.tag and studentTableModal.tag are separate files that I don't know about since I'm new. But it seems to me that the tip part could be merged into one file so that future changes for this would only have to be made in one file. 

But again, I understand that there might be some reason unbeknownst to me why this might be done on purpose.

Reply:
There may not be any good reason, other than the small differences between the two (e.g. the IDs are`studentTableWindow` vs `fsResultsTableWindow` and `detailsTable` vs `fsModalTable`). You are welcome to combine them into a single tag file.